tools/docs/build_docs.py

```diff
@@ -49,31 +49,46 @@
 
 flags.DEFINE_string(
     'git_branch',
-    default='master',
+    default=None,
     help='The name of the corresponding branch on github.')
 
+flags.DEFINE_string("output_dir", "/tmp/addons_api",
+                    "Where to output the docs")
+
+CODE_PREFIX_TEMPLATE = "https://github.com/tensorflow/addons/tree/{git_branch}/tensorflow_addons"
 flags.DEFINE_string(
-    'output_dir',
-    default='docs/api_docs/python/',
-    help='Where to write the resulting docs to.')
+    "code_url_prefix", None,
```

Comment:
How about putting `git_branch` and `code_url_prefix` together, and mark them as mutual exclusion？

Reply:
Sure. Fixed.